lib/firebase_id_token/configuration.rb

```diff
@@ -4,11 +4,22 @@ module FirebaseIdToken
   LIB_PATH = File.expand_path('../../', __FILE__)
 
   class Configuration
-    attr_accessor :redis, :project_ids, :certificates
+    attr_accessor :redis, :project_ids, :cache_store
 
     def initialize
       @project_ids = []
-      @certificates = FirebaseIdToken::Certificates
+    end
+
+    def certificates
+      klass
+    end
+
+    def certificates=(value)
+      @certificates = klass
```

Comment:
Shouldn't this be `@certificates = value`? Otherwise when you do `self.configuration.certificates = FirebaseIdToken::Testing::Certificates` in `self.test!` the testing certificates won't actually be set?

Reply:
I'm only looking at Firebase as a potential auth option, I'm not the right person to commit to something like this